runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -410,7 +411,7 @@ public static CrypticClue forText(String text)
 	{
 		for (CrypticClue clue : CLUES)
 		{
-			if (clue.text.equalsIgnoreCase(text))
+			if (clue.text.equalsIgnoreCase(Text.removeTags(text)))
```

Comment:
The removeTags call should be in ClueScrollPlugin along with the other code for cleaning of the clue scroll text.